Makefile

```diff
@@ -4,3 +4,14 @@ include boilerplate/generated-includes.mk
 boilerplate-update:
 	@boilerplate/update
 
+# ==> HACK ==>
+# We have to tweak the YAML to run this on 3.11.
+# We'll add a target to do that, then override the `generate` target to
+# call it.
+.PHONY: crd-fixup
+crd-fixup:
+	yq d -i deploy/crds/deadmanssnitch.managed.openshift.io_deadmanssnitchintegrations_crd.yaml spec.validation.openAPIV3Schema.type
```

Comment:
Do we have a standardized version of yq?

Reply:
It's in the [boilerplate image](https://github.com/openshift/boilerplate/blob/master/config/build.sh#L45-L50)